packages/data-service-generator/src/server/resource/resolver/to-one.template.ts

```diff
@@ -10,11 +10,15 @@ declare interface SERVICE {
 
 declare const ENTITY_NAME: string;
 declare const RELATED_ENTITY_NAME: string;
```

Comment:
@morhag90 let's just remove the unused RELATEDE_ENTITY_NAME like you did in the to-many.template.ts